internal/api/auth/auth.go

```diff
@@ -39,22 +37,9 @@ type Claims struct {
 	Nodes []string `json:"nodes"`
 }
 
-func (c Claims) validateExpiration(required bool) bool {
-	if c.Expiration == 0 && !required {
-		return true
-	}
-
-	now := time.Now().Unix()
-	return now <= c.Expiration
-}
-
-// Valid returns if the claim is valid (it hasn't expired yet).
-// It doesn't validate the content of the claim (
+// Valid validates the standard claims of the token.
 func (c Claims) Valid() error {
-	if !c.validateExpiration(false) {
-		return errors.New("token has expired")
-	}
-	return nil
+	return c.StandardClaims.Valid()
```

Comment:
I would expect that you can directly *inherit* this method from `StandardClaims`. Maybe you consider expanding it and that's why you decided to provide it on your own?